internal/operands/common-templates/resource.go

```diff
@@ -0,0 +1,129 @@
+package common_templates
+
+import (
+	"bytes"
+	templatev1 "github.com/openshift/api/template/v1"
+	"io"
+	"io/ioutil"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+const (
+	GoldenImagesNSname = "kubevirt-os-images"
+	bundleDir          = "data/common-templates-bundle/"
+	roleName           = "os-images.kubevirt.io:"
+	Version            = "v0.12.2"
+)
+
+func ClusterRoleName(role string) string {
```

Comment:
Maybe just have constants for both combinations? a function feels like an overkill

Reply:
Corrected now